packages/block-editor/src/components/index.js

```diff
@@ -36,6 +36,7 @@ export { default as ButtonBlockerAppender } from './button-block-appender';
 export { default as ColorPalette } from './color-palette';
 export { default as ColorPaletteControl } from './color-palette/control';
 export { default as ContrastChecker } from './contrast-checker';
+export { default as __experimentalDuotoneToolbar } from './duotone-toolbar';
```

Comment:
We've been moving away from "*Toolbar" components and replacing them with "*Control" components. (It's the same thing just removing the toolbar group wrapper)